docs/modules/ROOT/pages/quarkus-azure-storage-blob.adoc

```diff
@@ -9,7 +9,7 @@ If you want to use this extension, you need to add the `io.quarkiverse.azureserv
 
 For instance, with Maven, add the following dependency to your POM file:
 
-[source,xml,subs=attributes+]
+[source,xml]
```

Comment:
Looks like the value of ${project-version} is `0` which is defined in https://github.com/quarkiverse/quarkus-azure-services/blob/main/docs/modules/ROOT/pages/includes/attributes.adoc#LL1, and it's successfully substituted in the generated doc, see https://quarkiverse.github.io/quarkiverse-docs/quarkus-azure-services/dev/index.html.

I observed there is a duplicated [attributes.adoc](https://github.com/quarkiverse/quarkus-azure-services/blob/main/docs/templates/includes/attributes.adoc), not sure if removing this duplicated one can get rid of warnings?

Reply:
Ouch, that's right 🤦 I'll find another solution.